### PR TITLE
Added testRunner and implemented more tests

### PR DESCRIPTION
**Pull request by lodoyun**
This PR implements a test runner to make it easier to build variations of the basic test with different configurations.
To do so, it also adds code to pass the configuration options and adds checks for audio-only sessions.

### CODE REVIEW

File: KITE-Licode-Test/licodeTestRunner.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  let configNum = 1;
+  configs.forEach((config) => {
+    const configName = `${configPath}/config${configNum}.json`
+    config.writeToFile(configName);
+    runTestConfig(configName);
+    configNum++;
+  });
+};
+
+const firefoxAndChromeConfig = new KiteConfig('Firefox and Chrome Test Suite');
```
Comment (jcague) [inline KITE-Licode-Test/licodeTestRunner.js:39]: 👍 I really like this way of building tests!

File: KITE-Licode-Test/js/pages/MainPage.js [hunk 2]
```diff
@@ -58,6 +78,18 @@ class MainPage {
     return checked.result;
   }
 
+  async audioCheck(stepInfo, index) {
+    let checked; // Result of the verification
+    let i; // iteration indicator
+    let timeout = stepInfo.timeout;
+    stepInfo.numberOfParticipant = parseInt(stepInfo.numberOfParticipant); // To add the first video
+    console.warn(`Testing Audio with ${stepInfo.numberOfParticipant} participants, index`, index);
+    await TestUtils.waitForVideos(stepInfo, audios);
```
Comment (jcague) [inline KITE-Licode-Test/js/pages/MainPage.js:87]: np: shouldn't this be callign the new `waitForAudios`?
```suggestion
    await TestUtils.waitForAudios(stepInfo, audios);

```

File: KITE-Licode-Test/js/pages/MainPage.js [hunk 3]
```diff
@@ -58,6 +78,18 @@ class MainPage {
...
   }
 
+  async audioCheck(stepInfo, index) {
+    let checked; // Result of the verification
+    let i; // iteration indicator
+    let timeout = stepInfo.timeout;
+    stepInfo.numberOfParticipant = parseInt(stepInfo.numberOfParticipant); // To add the first video
+    console.warn(`Testing Audio with ${stepInfo.numberOfParticipant} participants, index`, index);
+    await TestUtils.waitForVideos(stepInfo, audios);
+    // Waiting for all the videos
```
Comment (jcague) [inline KITE-Licode-Test/js/pages/MainPage.js:88]:
```suggestion
    // Waiting for all the audios

```

File: KITE-Licode-Test/licodeTestRunner.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+testSinglePCSimulcast.setSimulcast(true);
+testSinglePCSimulcast.setSinglePC(true);
+onlyChromeConfig.addTest(testSinglePCSimulcast);
+
+
+configs.push(firefoxAndChromeConfig);
+configs.push(onlyChromeConfig);
+runAllTests();
+
+
```
Comment (jcague) [inline KITE-Licode-Test/licodeTestRunner.js:69]: np: too many empty lines?

**Review by jcague — APPROVED** [review]
this looks great! I just left some minor comments

File: KITE-Licode-Test/licodeTestRunner.js (hunk 4 above)
Comment (lodoyun) [reply KITE-Licode-Test/licodeTestRunner.js:69]: 👍

**Pull request merged**